code/modules/power/lighting.dm

```diff
@@ -653,3 +653,14 @@
 		force = 5
 		playsound(src.loc, 'sound/effects/Glasshit.ogg', 75, 1)
 		update()
+
+
+/obj/machinery/light/floor
```

Comment:
We don't have floor lighting and I thought it would be nice for more lighting options?